app/graphql/resolvers/customer/query.js

```diff
@@ -1,8 +1,13 @@
 import { ruralPaymentsPortalCustomerTransformer } from '../../../transformers/rural-payments-portal/customer.js'
+import { Query as CustomerApplicationQuery } from './applications.js'
 
 export const Query = {
   async customer (__, { id }, { dataSources }) {
     const response = await dataSources.ruralPaymentsPortalApi.getCustomerByCRN(id)
-    return ruralPaymentsPortalCustomerTransformer(response)
+
+    return {
+      ...ruralPaymentsPortalCustomerTransformer(response),
```

Comment:
Think we should either mode this transformer to the resolver or you should match the way it is done